.changesets/maint_garypen_3533_istio_warn.md

```diff
@@ -0,0 +1,9 @@
+### Add a warning if we think istio-proxy injection is causing problems ([Issue #3533](https://github.com/apollographql/router/issues/3533))
+
+We have encountered situations where the injection of istio-proxy in a router pod (executing in kubernetes) causes strange networking errors during uplink retrieval.
+
+The root cause of the issue is that the router is executing and attempting to retrieve uplink schemas whilst the istio-proxy is modifying network configuration at the same time.
+
+This new warning message will direct users to information which should help them to configure their kubernetes cluster or pod to avoid this problem.
```

Comment:
```suggestion
This new warning message directs users to information which should help them to configure their Kubernetes cluster or pod to avoid this problem.
```